tooling/github-strategy.md

```diff
@@ -0,0 +1,92 @@
+- [Conclusion: High Level Guideline](#conclusion-high-level-guideline)
+
+## Balancing Security & InnerSource: Who Can See What
+
+The section explores the delicate balance between security and collaboration in the context of InnerSource projects. Within an InnerSource project, transparency plays a vital role in fostering collaboration and encouraging participation. Ideally, the project should be structured to allow as many individuals as possible to contribute, ensuring that the barriers to participation are kept low. However, it's important to acknowledge that certain constraints and considerations may prevent making everything openly accessible within the company.
+
+To strike a balance between encouraging InnerSource participation and respecting the company's constraints, it becomes necessary to carefully consider different levels of code sharing and establish appropriate Source Code Management (SCM) practices. This allows for the implementation of controlled visibility and access mechanisms, ensuring that sensitive or proprietary code remains protected while still enabling collaboration and knowledge sharing.
+
+By defining these various levels of sharing, organizations can customize their SCM setup to accommodate their specific needs. This may involve setting up different repositories or access controls based on the sensitivity of the code, project type, or individual roles within the organization. Through thoughtful planning and implementation, companies can create an InnerSource environment that maximizes participation while upholding necessary restrictions.
+
+By effectively managing transparency and sharing levels, organizations can foster a culture of InnerSource collaboration that strikes the right balance between openness and the company's specific constraints. This approach enables individuals to contribute their expertise, fosters innovation, and strengthens the overall development process within the organization.
+
+## Security-First Perspective and InnerSource-first Perspective
+
+There are two perspectives presented in the table: the Security-First Perspective and the InnerSource-First Perspective. By considering these two perspectives, companies can address the risks associated with sensitive code leaving the organization while also fostering collaboration and sharing of code within the company for increased productivity and innovation.
+
+| ​ | Security-First Perspective​ | InnerSource-first perspective​ |
+| --- | --- | --- |
+| Concerned with What Risk?​ | Sensitive code gets outside Company. ​ | No effective way for anyone at company to search, discover, or evaluate code for use created outside their org.​ |
+| Absolute positions for each perspective​ | All repositories should be their own silo. Access to know each repo exists only individually granted by upper management.​ | All repositories should be visible to all FTEs.​ |
```

Comment:
From @dellagustin  ([The original review comment](https://docs.google.com/document/d/1lPR90dIiWKlsNlV46xhGkR9pP0GNYmeAB7xOk-_MAYo/edit?disco=AAAA1q2rBbk))

Maybe add here something like "in a need-to-know basis", it is a term often used in this context and for this absolute position.